dpl_htaccess

```diff
@@ -0,0 +1,12 @@
+# dlang.org .htaccess file (local Apache configuration)
+
+# Enable mod_rewrite
+RewriteEngine On
+
+# Indicate that this directory maps to the root
+# directory of the domain.
+RewriteBase /
```

Comment:
BTW another thing, shouldn't we redirect only 404s? What if a new identifier appears that by the new convention has the same URL as a redirect?